src/index.js

```diff
@@ -404,9 +428,16 @@ class ProceduresVisualizer extends GenericVisualizer {
   static defaultProps = {
     title: 'Procedures',
     columns: [
-        { title: 'SNOMED', versions: '*', getter: c => c.code.coding[0].code },
-        { title: 'Procedure', versions: '*', getter: c => c.code.coding[0].display },
-        { title: 'Date Performed', versions: '*', format: 'dateTime', getter: c => c.performedPeriod.start }
+      { title: 'SNOMED', versions: '*', getter: p => p.code.coding[0].code },
+      { title: 'Procedure', versions: '*', getter: p => p.code.coding[0].display },
+      { title: 'Date Started', versions: '*', format: 'dateTime', getter: p => p.performedPeriod.start },
+      { title: 'Date Completed', versions: '*', format: 'dateTime', getter: p => p.performedPeriod.end },
+      { title: 'Date Performed', versions: '*', format: 'dateTime', getter: p => p.performedDateTime },
+      { title: 'ID', versions: '*', getter: p => p.id },
+      { title: 'Recorded', versions: '*', format: 'dateTime', getter: p => p.extension.recorded },
+      { title: 'Reason', versions: '*', getter: p => `${p.reasonCode.coding[0].code}, ${p.reasonCode.coding[0].display}` },
```

Comment:
I see this approach of `${code} ${display}` used in a couple places. How would you feel about adding a new format type "code" (or something) in FORMATTERS above,
```js
code: (code) => `${code.code} ${code.display}`,
```
 so here we could just do:

```suggestion
      { title: 'Reason', versions: '*', format: 'code', getter: p => p.reasonCode.coding[0] },
```

Reply:
This cleans things up so much- great idea!